code/unclassified/minSubarraySizeWithDegree/minSubarraySizeWithDegree.cpp

```diff
@@ -0,0 +1,66 @@
+#include <bits/stdc++.h>
```

Comment:
Is is not preferable to use this header as it is not supported by all compilers.

Reply:
Ok, makes sense